package.json

```diff
@@ -69,13 +69,15 @@
     "husky": "=4",
     "lint-staged": "^11.2.6",
     "node-sass": "^5.0.0",
-    "postcss": "^8.2.15",
+    "postcss": "^8.4.5",
     "postcss-cli": "^8.3.1",
-    "postcss-discard-comments": "^4.0.2",
+    "postcss-discard-comments": "^5.0.1",
+    "postcss-html": "^1.3.0",
```

Comment:
![image](https://user-images.githubusercontent.com/36186912/148910165-a8927425-dd82-488d-9b73-b9e7bef9a740.png)
That why we need `postcss-html` and `stylelint-config-html`.

Reply:
Do we really need postcss-html? Stylelint doesn't help in html files

Reply:
So why we have this? 😀
![image](https://user-images.githubusercontent.com/36186912/148956826-efe564a1-c6bb-4727-b5ab-2f7a83e23db8.png)

Reply:
I dont know, I'm saying maybe we should remove it instead of adding a new dependency

Reply:
Removed